core/arch/arm/include/mm/core_mmu.h

```diff
@@ -86,6 +86,7 @@
  * MEM_AREA_IO_SEC:   Secure HW mapped registers
  * MEM_AREA_RES_VASPACE: Reserved virtual memory space
  * MEM_AREA_TA_VASPACE: TA va space, only used with phys_to_virt()
+ * MEM_AREA_TEE_RAM_UNCACHED:  Same as MEM_AREA_TEE_RAM, but cache disabled.
```

Comment:
Yes. Or, since such memory will be needed for "Power System Control" sequences, you could name it `PSC_RAM` (or split _RAM/_ROM/_XROM if required).

Note that I don't think splitting RAM into RAM/ROM/XROM is bullet proof. Maybe it's something you could go into later on, once the basic functionality of you suspend/resume sequence are validate.